include/cglass/crosslink_manager.hpp

```diff
@@ -44,6 +44,8 @@ class CrosslinkManager {
   const double GetDrMax();
   void ReadInputs();
   void InsertCrosslinks();
+  void BeginWithCrosslinks();
+  void BeginWithCrosslinkers();
```

Comment:
This method is not initialized. Only BeginWithCrosslinks is. CrosslinkSpecies has a BeginWithCrosslinkers method so I think you should keep that nomenclature consistent or change it more to something like BeginSpeciesWithCrosslinkers to make it less confusing and prevent ambiguity in ctag searches. I might also suggest using "Init" instead of "Begin" to keep conventions consistent.

Reply:
Thanks for the suggestion, I renamed BeginWithCrosslinkers to BeginSpeciesWithCrosslinks. 
I don't understand why changing Begin to Init would be consistent with conventions.